airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/DefaultJobPersistence.java

```diff
@@ -917,20 +919,32 @@ private static Job getJobFromRecord(final Record record) {
     return new Job(record.get(JOB_ID, Long.class),
         Enums.toEnum(record.get("config_type", String.class), ConfigType.class).orElseThrow(),
         record.get("scope", String.class),
-        Jsons.deserialize(record.get("config", String.class), JobConfig.class),
+        parseJobConfigFromString(record.get("config", String.class)),
```

Comment:
how would we have a jobconfig with a v0 schema? Wouldn't we upgrade the schema before creating the jobconfig?

Reply:
Since we're updating on the fly, there is no migrations of the DB. All jobs that were created before the upgrade will have v0 data.